api/v1beta1/ibmpowervs_conversion.go

```
@@ -141,3 +147,45 @@ func (dst *IBMPowerVSImageList) ConvertFrom(srcRaw conversion.Hub) error {
 
 	return Convert_v1beta2_IBMPowerVSImageList_To_v1beta1_IBMPowerVSImageList(src, dst, nil)
 }
+
+func Convert_v1beta1_IBMPowerVSMachineSpec_To_v1beta2_IBMPowerVSMachineSpec(in *IBMPowerVSMachineSpec, out *infrav1beta2.IBMPowerVSMachineSpec, s apiconversion.Scope) error {
+	out.SystemType = in.SysType
+	out.Processors = intstr.FromString(in.Processors)
+
+	memory, _ := strconv.ParseFloat(in.Memory, 64)
```

why are we ignoring error here?

---

Fixed.